packages/web/src/index.html

```diff
@@ -269,6 +269,13 @@
           required
         ></gcds-input>
 
+        <gcds-date-input
+          legend="Date input"
```

Comment:
Sorry, nitpicky but can we add a hint here as well, please? Seems like all the other form fields in the example use a hint.